### ISSUE TITLE
Number of thumbs up/down not showing in recent messages

### ISSUE THREAD

**Issue by JediMaster25** [LAION-AI/Open-Assistant#1367]
After #1303 I can no longer see the thumbs up/down on my recent messages. And I can't vote on them either so they need to show by default.

**Comment by aayojezuzchrezt** [comment]
Yeah, I think it's important to know if users like or don't your answers, so you can correct them in future

**Comment by notmd** [comment]
@fozziethebeat can you leave your thoughts here? My idea is only hidden on the task page only.

**Comment by fozziethebeat** [comment]
I think it makes sense to generally hide the votes for regular users in the message or tasks page (until they make a vote).  

But for the "Your Messages" sub-section for the messages page, we could show a read-only report of the reactions so that users know how their messages are getting rated.  Does that need to use the same emoji buttons embedded in the message? only if convenient.  We could also add a different wrapper around each message with reaction details (and maybe future stuff we want to tell message authors at some point).
